code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -1273,14 +1263,14 @@ GLOBAL_LIST_INIT(available_specialist_sets, list("Scout Set", "Sniper Set", "Dem
 		/obj/item/clothing/glasses/hud/health,
 		/obj/item/clothing/under/marine/corpsman,
 		/obj/item/clothing/shoes/marine/full,
+		/obj/item/clothing/head/helmet/marine/corpsman,
```

Comment:
duplicated
```suggestion
```

Reply:
@SplinterGP  My bad, I stated a review some time ago, but forgot to hit submit. I'm not trying to step on your toes.

Reply:
Fixed